deploy/role.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: registration-service
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - usersignups
+  verbs:
+  - create
+  - get
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+    - masteruserrecord
+  verbs:
+    - get
```

Comment:
same here: no need to `list`?

Reply:
Correct. Will add list later if we need. But for now let’s grand minimum permissions we need.

Reply:
ok, sounds reasonable/pragmatic to only address today's requirements.